react/features/recording/components/LiveStream/StreamKeyForm.web.js

```diff
@@ -109,7 +78,7 @@ class StreamKeyForm extends Component {
      * @returns {void}
      */
     _onOpenHelp() {
-        window.open(this.props.helpURL, 'noopener');
```

Comment:
Conceptually, should/can helpURL be passed in as prop?

Reply:
Absolutely! I just favour more self-containing components, but it's definitely feasible. Would you prefer that?

Reply:
I did prefer it but I'd rather you go forward with your preference here because you'll be continuing work around this feature and it's not a significant enough change for concern. If I really care I can swoop in afterwards and mess up your changes to my style.

Reply:
Haha, okay, feel free to do it :)